arcus_orientation.md

```diff
@@ -757,11 +766,11 @@ To give researchers peace of mind, Arcus provides a tool that allows you to inst
 
 You can reach the Install Tool from your lab dashboard, under Tools.
 
-![An Arcus Lab dashboard, showing the Tools section with "Install packages, downloads, and more" at the end of the list.](media/install_dashboard.png)
+![An Arcus Lab dashboard, showing the Arcus Lab Tools section with "New Install Tool" at the end of the list.](media/install_dashboard.png)
```

Comment:
This image isn't showing up. I think the file is accidentally named install_dashbaord.png

Reply:
just pushed a change to fix the name -- that being said, we may need to take this screenshot again anyways, once the name is fixed in the dev labs. (Or do what Joy suggested and edit it somehow).